src/features/goblins/bank/lib/bankUtil.test.ts

```diff
@@ -302,7 +302,7 @@ describe("canWithdraw", () => {
     expect(enabled).toBeFalsy();
   });
 
-  it("prevent a user to withdraw kuebiko while they have seeds", () => {
+  it("prevents a user to withdraw kuebiko while they have seeds", () => {
```

Comment:
"prevents" wasn't bad grammar but "to withdraw" should be "from withdrawing"

Reply:
Yeah true, I can update this! Thanks for the feedback.